canonical-web-and-design/ubuntu.com
# Local version of rtp script with jquery off

## Done

- added a local version of the rtp script with loadJquery to false

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: http://0.0.0.0:8001/static/js/rtp.js
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- This is for a performance test
